.env

```diff
@@ -1,2 +1,2 @@
-PORT=8083
+PORT=8014
 DEBUG=False
```

Comment:
Are this DEBUG and DEBUG_MODE in configmap the same thing?

Reply:
Updated configmap to be consistent with this.